src/Core/OrganizationFeatures/AuthorizationHandlers/GroupAuthHandler.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Bit.Core.Context;
+using Bit.Core.Entities;
+using Bit.Core.Enums;
+using Microsoft.AspNetCore.Authorization;
+
+namespace Bit.Core.OrganizationFeatures.AuthorizationHandlers;
+
+class GroupAuthHandler : AuthorizationHandler<GroupOperationRequirement, Group>
```

Comment:
nit: I'd prefer the full `GroupAuthorizationHandler` it's spelled out for properties in our code styling guide but I'd argue it should extend to classes and especially classes that only need to be referenced one other time as service registration.